web/src/designer/fields.tsx

```diff
@@ -285,6 +285,7 @@ const fields: {[key: string]: FieldType} = {
       relation_type: 'faims-core::Child',
       multiple: false,
       allowLinkToExisting: true,
+      showCreateAnotherButton: true,
```

Comment:
Can this be 'hideCreateAnotherButton' so that we can maintain the current behaviour by default and turn it off if required.